src/status_im2/contexts/profile/login/events.cljs

```diff
@@ -151,12 +162,16 @@
 
 (rf/defn get-auth-method-success
   {:events [:profile.login/get-auth-method-success]}
-  [{:keys [db]} auth-method]
+  [{:keys [db] :as cofx} auth-method key-uid]
```

Comment:
I don't see `cofx` being used here, maybe I am missing something though

Reply:
wasn't used and found another unused one in the same file